src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowWalker/Pages/SettingsPage.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CmdPal.Ext.WindowWalker.Helpers;
+using Microsoft.CmdPal.Ext.WindowWalker.Properties;
+using Microsoft.CmdPal.Extensions;
+using Microsoft.CmdPal.Extensions.Helpers;
+
+namespace Microsoft.CmdPal.Ext.WindowWalker.Pages;
+
+internal sealed partial class SettingsPage : FormPage
+{
+    private readonly Settings _settings;
+    private readonly SettingsManager _settingsManager;
+
+    public override IForm[] Forms()
+    {
+        var s = _settings.ToForms();
+        return s;
+    }
+
+    public SettingsPage(SettingsManager settingsManager)
+    {
+        Name = "Settings Page NEED TO MAKE RESOURCES FOR THIS";
```

Comment:
TODO

Reply:
🫨 - that is on me. 

I think this will become natural once we figure out #194, because then every title can just be "Settings"... which I already starting doing elsewhere.

Reply:
Made it "Settings" in resources. Done.